pkgs/misc/jackaudio/default.nix

```diff
@@ -48,7 +49,7 @@ stdenv.mkDerivation (finalAttrs: {
 
   prePatch = ''
     substituteInPlace svnversion_regenerate.sh \
-        --replace /bin/bash ${bash}/bin/bash
+        --replace /bin/bash ${buildPackages.bash}/bin/bash
```

Comment:
If it's a shebang then
`patchShebangs --build svnversion_regenerate.sh`

Reply:
Done